### DISCUSSION TITLE
I config the minio connect to ldap but it seem get some error.

### DISCUSSION THREAD

**Discussion by andyzheung** [minio/minio#15880] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/22912347/196148938-5d917eb3-4c6b-40ab-9980-7c89d2c5a38b.png)
I can't access through the web through the old accesskey and secretkey, and also can't use ldap.
can anyone help me, I just want to come back to the old access way. So that I can use it..

**Comment by vadmeste** [comment, marked as answer]
@andyzheung for now, you can disable all LDAP configuration and use MINIO_ROOT_USER and MINIO_ROOT_PASSWORD to access to Console.

What is your MinIO version ?

**Comment by andyzheung** [comment]
minio-4.0.2

1、How to disable all the LDAP? I use helm chart deploy minio on k8s.
2、If I want to use LDAP,  the user I create before can't be use?
That means I want to use LDAP and can also create the user by admin

**Reply by suhailsid** [reply]
can we not use both LDAP and local user parallelly as per the latest release it support multiple IDP authentication, can we use this for LDAP and Local Miniouser ?

**Comment by andyzheung** [comment]
I have solve the minio problem..
But our ldap user can't use mc cli to access the minio?

**Reply by vadmeste** [reply]
> But our ldap user can't use mc cli to access the minio?

You can create a service account for that LDAP user and use it with 'mc'. (mc admin user svcacct add -h)
